openapi/publicapi_spec.json

```diff
@@ -1790,6 +2199,24 @@
           }
         }
       },
+      "PaymentInstrument" : {
+        "type" : "object",
+        "properties" : {
+          "CardDetails" : {
+            "$ref" : "#/components/schemas/CardDetails"
+          },
+          "created_date" : {
+            "type" : "string",
+            "description" : "The date and time you created this payment instrument. This value uses Coordinated Universal Time (UTC) and ISO 8601 format – `YYYY-MM-DDThh:mm:ss.sssZ`.",
+            "example" : "2022-07-08T14:33:00.000Z"
+          },
+          "type" : {
+            "type" : "string",
+            "description" : "The type of payment instrument.",
+            "example" : "card"
```

Comment:
We could maybe put an `enum` of possible values for this. The only possible value is "card"